src/database/standalone-sqlite.ts

```
@@ -3261,10 +3342,26 @@ if (!isMainThread) {
           const isHashBlocked = worker.isHashBlocked(args[0]);
           parentPort?.postMessage(isHashBlocked);
           break;
+        case 'isNameBlocked':
+          const isNameBlocked = worker.isNameBlocked(args[0]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)